src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/global_navitagtion/OnSiteNotifications.java

```diff
@@ -0,0 +1,86 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00120)